x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -161,27 +166,41 @@ export const ActionsConnectorsList: React.FunctionComponent = () => {
         );
 
         const link = (
-          <EuiLink
-            data-test-subj={`edit${item.id}`}
-            onClick={() => editItem(item, EditConectorTabs.Configuration)}
-            key={item.id}
-            disabled={actionTypesIndex ? !actionTypesIndex[item.actionTypeId]?.enabled : true}
-          >
-            {value}
-          </EuiLink>
+          <>
+            <EuiLink
+              data-test-subj={`edit${item.id}`}
+              onClick={() => editItem(item, EditConectorTabs.Configuration)}
+              key={item.id}
+              disabled={actionTypesIndex ? !actionTypesIndex[item.actionTypeId]?.enabled : true}
+            >
+              {value}
+            </EuiLink>
+            {item.isMissingSecrets ? (
+              <EuiIconTip
+                iconProps={{ 'data-test-subj': `missingSecrets_${item.id}` }}
+                type="alert"
+                color="warning"
+                content={i18n.translate(
+                  'xpack.triggersActionsUI.sections.actionsConnectorsList.connectorsListTable.columns.actions.missingSecretsDescription',
+                  { defaultMessage: 'Sensitive information is not imported' }
```

Comment:
Maybe `Sensitive information was not imported` instead of is? `is` sounds off to me, but maybe that's just me.

@gchaps

Reply:
I think @ymao1 is right because the importing of information has already taken place.